### DISCUSSION TITLE
Support for multiple vercel.json?

### DISCUSSION THREAD

**Discussion by gniquil** [vercel/vercel#4672] [Help] [answered]
For my project, I need to use proxy to integrate my legacy server. However, I need to proxy to different domains depending on whether I'm doing production build vs preview build. Thus I would love to be able to configure say in the dashboard for preview deploys a different file 'vercel.preview.json' that defines a different set of rewrite rules. 

Or is there another way to do that?

**Comment by styfle** [comment, marked as answer]
Hi @gniquil 

Looks like this is the same question as #4677 so lets continue the discussion there.
